client/payment-methods/index.js

```diff
@@ -47,27 +51,16 @@ function PaymentMethodsDropdownMenu() {
 				{
 					title: __( 'Provide Feedback', 'woocommerce-payments' ),
 					icon: 'megaphone',
-					onClick: () => console.log( 'Provide Feedback' ),
+					onClick: () => setOpenModal( 'survey' ),
 				},
 				{
 					title: 'Disable',
 					icon: trash,
-					onClick: () => setIsUpeEnabled( false ),
+					onClick: () => setOpenModal( 'disable' ),
 				},
 			] }
 		/>
 	);
-}
-
-const UpeDisableError = () => {
```

Comment:
Was it intended to also remove this?

Reply:
Yep! That isn't a Notice anymore - the Disable Modal itself will dispatch an error notice with `dispatch( 'core/notices' ).createErrorNotice()`.